### ISSUE TITLE
Legislation - Replace element "em" by "cite"  (accessibility)

### ISSUE THREAD

**Issue by duboisp** [wet-boew/wet-boew-styleguide#75]
Hi,

It was brought to my attention that 

As per WCAG 2.0, when coding names of Acts and legislation, the `<cite>` tag needs to be used (when providing a citation or a reference to other sources).

The `<em>` tag should only be used to indicate emphasis.

However, we noticed that the Writing Guide, [Legislation](http://wet-boew.github.io/wet-boew-styleguide/new/writing/formatting-en.html#legislation) was coded using the `<em>` tag for "Income Tax Act" instead of the `<cite>` tag.  We think that this should be changed to `<cite>` to meet accessibility requirements.

But before to change it, I observed that text into `<cite>` are not rendered in italics. Is that should be his style? Because for WET 3.1 the text in `<cite>` are rendered in italics.

:smile:

**Comment by sviens** [comment]
Doing some housekeeping. Can this issue be closed?
cc/ @duboisp @pjackson28 @cfarquharson @davidasselin

**Comment by Wenifred** [comment]
The latest WET (v4.0.11) renders the `<cite>` element in italics. 
It's a change in Bootstrap: https://github.com/twbs/bootstrap/commit/9677c95d83f361b7c3a43e56ef89590305c84905

Do we still code it like: `<cite><i>Income Tax Act</i></cite>`
What about if it is an author and not a publication, act..etc.?

**Comment by RobJohnston** [comment]
The page referenced by @duboisp is now a dead link.  Could it now be https://www.tbs-sct.gc.ca/hgw-cgf/oversight-surveillance/communications/csc-grc-eng.asp#wp2-7?  

If so, there is no `<cite>` tag in the example anymore and there's a whole big discussion about italics in https://github.com/wet-boew/wet-boew/issues/7631.  Time to close this?
